executers/utils/gopath_unix.go

```diff
@@ -0,0 +1,10 @@
+// +build linux darwin freebsd
```

Comment:
I think it is best to have the code check for the OS, instead of having different code built of each OS.
Using the option of building code per OS should be used when there's no way of doing the later option. For example, if different packages should be important for each OS. 
Avoiding the above approach may also allow you not add a new *gopath_unix.go* file.

Reply:
Done